CHANGELOG.md

```diff
@@ -11,6 +11,22 @@ Some experimental features can be enabled / disabled using [Feature Gates](docs/
 
 ## Unreleased
 
+## 0.9.3 - 2020-09-03
+
+### Changed
+
+- Rename *internal* API group from "networking.antrea.tanzu.vmware.com" to "controlplane.antrea.tanzu.vmware.com". ([#1147](https://github.com/vmware-tanzu/antrea/pull/1147), [@jianjuns])
+   * This API is served by the Antrea Controller and consummed by Agents (directly) and antctl (through the K8s apiserver using an APIService)
+   * Antrea Controller deletes the previous APIService on startup to avoid issues (e.g. with namespace deletion)
+   * Users should not be affected unless they consummed the API directly which is unlikely; during upgrade from a previous version, NetworkPolicy enforcement may be disrupted until the upgrade is complete
+- Upgrade OVS version to 2.14.0 to pick up some recent patches. ([#1121](https://github.com/vmware-tanzu/antrea/pull/1121), [@lzhecheng])
+
+### Fixed
+
+- Fix IPsec support which was broken after updating the base distribution to Ubuntu 20.04 for the Antrea Docker image, as this update introduced a more recent version of [strongSwan]. ([#1184](https://github.com/vmware-tanzu/antrea/pull/1184) [#1191](https://github.com/vmware-tanzu/antrea/pull/1191), [@jianjuns])
```

Comment:
not sure if this should go in the notes but you also added kustomize version commit to the release.. also since you added it do you want to include the gitignore commit or its not really needed since it mostly affects dev activities?

Reply:
I added the gitignore commit just in case, but there should be no development in this branch.
I did not include my kustomize patch in the notes since I feel it does not impact users, even if it changed the yaml manifests a bit

Reply:
makes sense